jsk_topic_tools/src/connection_based_nodelet.cpp

```diff
@@ -52,6 +60,13 @@ namespace jsk_topic_tools
     }
   }
 
+  void ConnectionBasedNodelet::warnNeverSubscribedCallback(const ros::WallTimerEvent& event)
+  {
+    if (!ever_subscribed_) {
+      NODELET_WARN("'%s' subscribes topics only with child subscribers. (`always_subscribe: true` to force it to subscribe)", nodelet::Nodelet::getName().c_str());
```

Comment:
always_subscribe requires child class implementation and I didn't implemented that features in the most of nodelets.
Can you remove description about always_subscribe? it may confuse users.